libs/chain-signatures/contract/tests/updates.rs

```diff
@@ -208,13 +202,13 @@ async fn test_invalid_contract_deploy() {
     let state: ProtocolContractState = execution.json().unwrap();
     dbg!(state);
 }
-// todo: fix this test
+
 #[tokio::test]
 async fn test_propose_update_contract_many() {
     let (_, contract, accounts, _) = init_env_secp256k1(1).await;
     dbg!(contract.id());
 
-    const PROPOSAL_COUNT: usize = 10;
+    const PROPOSAL_COUNT: usize = 5;
```

Comment:
I randomly got "Size of the recorded trie storage proof has exceeded ... yada yada", so I decided to drop the number by a little bit

Reply:
Ah I see that's probably an underlying blockchain/test framework limitation.